anitya/lib/model.py

```diff
@@ -439,6 +439,19 @@ def by_name_and_homepage(cls, session, name, homepage):
         return query.first()
 
     @classmethod
+    def by_version_url(cls, session, version_url):
+        try:
+            query = session.query(
+                cls
+            ).filter(
+                cls.version_url == version_url
+            )
+            return query.one()
```

Comment:
I agree that there will be more then one result in this. However I created this function because I need to query database whether there is `name` defined for `version_url`. When I am going to add new maven project, I have no information about `name`. All I have is `version_url` which consists of `groupId:artifactId`. So I will try whether there is package with such `version_url` and if I use that `name`.

I know that this query returns more results, but they should vary just in version. I will update it to return all matching projects and I will get just one from which I obtain name.